sentinel-core/src/test/java/com/alibaba/csp/sentinel/util/function/Tuple2Test.java

```diff
@@ -0,0 +1,14 @@
+package com.alibaba.csp.sentinel.util.function;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class Tuple2Test {
+
+    @Test
+    public void testSwap() {
+        Tuple2<Integer, Integer> tupleOld = new Tuple2(1, 2);
+        Tuple2<Integer, Integer> tupleNew = tupleOld.swap();
+        Assert.assertEquals("Tuple2{r1=2, r2=1}", tupleNew.toString());
```

Comment:
Maybe it's better to compare the r1/r2 value directly?

Reply:
done